src/gui/proxyauthhandler.cpp

```diff
@@ -167,11 +166,25 @@ bool ProxyAuthHandler::getCredsFromDialog()
     return false;
 }
 
-bool ProxyAuthHandler::getCredsFromKeychain()
+template<class T, typename PointerToMemberFunction>
+void ProxyAuthHandler::execAwait(const T *sender,
+                                 PointerToMemberFunction signal,
+                                 int &counter,
+                                 const QEventLoop::ProcessEventsFlags flags)
 {
-    using namespace QKeychain;
+    if(!sender)
```

Comment:
Missing space before opening parenthesis, also we should probably use the curly braces every time (still up for discussions of course).

Reply:
Well the curly brace thing would really have to be considered for a lot of places in the codebase then.

Reply:
Sure